knowledge-base/editor-add-icons.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Add Icons in Editor
+description: How to allow inserting <i> tags in the Editor for Blazor to render custom icons in the Editor content?
+type: how-to
+page_title: Add Icons in Editor
+slug: editor-kb-add-icons
+position: 
+tags: editor, blazor, icon, schema, prosemirror
+ticketid: 
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Editor for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+This KB article answers the following questions:
+
+* How to add custom icons in the Editor content? When adding an icon such as `<i class="fa-light fa-envelope">`, the icon's HTML disappears upon saving.
+* How can I add an `<i>` element inside the Editor?
+
+## Solution
+
+The built-in ProseMirror schema that the Editor uses does not support an `<i>` tag. To allow adding icons in the Editor content:
+
+1. [Modify the ProseMirror schema]({%slug editor-modify-default-schema%}) to include a node for the `<i>` element.
+2. Ensure the CSS classes for the icons are applied. The approach varies depending on the [edit mode of the Editor]({%slug editor-edit-modes-overview%}):
```

Comment:
Our customers know that. The tricky part that is in our support scope is how to inject the stylesheet. So:

```suggestion
2. Ensure the required font icon stylesheets can affect the Editor content. The approach varies depending on the [edit mode of the Editor]({%slug editor-edit-modes-overview%}):
```